src/pomelo_fmt_plug.c

```diff
@@ -25,20 +25,19 @@ john_register_one(&fmt_pomelo);
 #include "formats.h"
 #include "params.h"
 #include "options.h"
+#include "simd-intrinsics.h"
 
 #define FORMAT_LABEL            "pomelo"
 #define FORMAT_NAME             ""
 #define FORMAT_TAG              "$pomelo$"
 #define TAG_LENGTH              sizeof(FORMAT_TAG) - 1
 
-#if !defined(JOHN_NO_SIMD) && defined(__AVX2__)
-#define ALGORITHM_NAME          "POMELO 256/256 AVX2 1x"
-#elif !defined(JOHN_NO_SIMD) && defined(__SSE2__)
-#define ALGORITHM_NAME          "POMELO 128/128 SSE2 1x"
+#if defined BITS && defined SIMD_TYPE
+#define ALGORITHM_NAME          BITS " " SIMD_TYPE " 1x"
 #elif !defined(USE_GCC_ASM_IA32) && defined(USE_GCC_ASM_X64)
```

Comment:
This looks like erroneous copy-paste from the "gost" format, and needs a separate fix.

Reply:
Isn't gost and stribog closely related formats? I have no idea what is correct and what is wrong.

Reply:
gost and stribog are closely related formats, but we're commenting on pomelo here. Apparently, pomelo_fmt_plug.c was copied from gost's when it was being created, perhaps because both hash types were new for jumbo during the same contest (Hash Runner?)

Reply:
Ah sorry, this is pomelo and not stribog

Reply:
BTW, we could consider dropping POMELO. It's a PHC finalist, and was used as such during a contest back then, but PHC is now over and I don't think anyone uses POMELO anywhere.

Reply:
> we could consider dropping POMELO.

I am a big fan of removing code. But, I think magnum should make a 'fake' release to save a zip file containing all these files. 
- 1.9.0J1-RC is a good/acceptable name;

Reply:
So I would just tag the commit prior to this one, named `1.9.0-Jumbo-1-RC` right? Or do you mean something more with "fake release"?

Reply:
I prefer an official zip file with a description (so, a fake release). You can name it "shrink" or anything like that.

Reply:
If we do that, then maybe tag the revision from just prior to recent dropping of `nt-old` and `*-ng` formats.

Reply:
So, a `Shrink-To-1.9.0-Jumbo-1` release could be a resource to anyone looking for code examples these formats provide.

Reply:
> If we do that, then maybe tag the revision from just prior to recent dropping of `nt-old` and `*-ng` formats.

I rebased the commits so we still fix Pomelo's algo name right before dropping it in a separate commit. And all drop commits now come last in this PR.

Reply:
I'll tag it once I know the final git hash - I think GitHub will then provide a zip for that exact version.

Reply:
> > If we do that, then maybe tag the revision from just prior to recent dropping of `nt-old` and `*-ng` formats.
> 
> I rebased the commits so we still fix Pomelo's algo name right before dropping it in a separate commit. And all drop commits now come last in this PR.

That's not reflected in the web gui... but it looks like this here:
```
commit e13af57c0b44f98a27c990f08c5870bd355beb11 (HEAD -> bleeding-jumbo, origin/botcheck)
Author: magnum <john.magnum@hushmail.com>
Date:   Mon May 6 21:37:56 2019 +0200

    Drop Pomelo format, it's obsolete and probably not used anywhere.
    See #3881

commit 3ca950d08d0e6b36108b805cf6677a36cfdfc877
Author: magnum <john.magnum@hushmail.com>
Date:   Mon May 6 02:24:19 2019 +0200

    Drop Alain's good old NT format. It's finally obsolete. #3881

commit 3cfe77507c3d86701715dec6007b90d11b2e2ff5
Author: magnum <john.magnum@hushmail.com>
Date:   Mon May 6 01:45:49 2019 +0200

    Drop the "-ng" SIMD formats. They are obsolete. See #3881

commit 91520a994d1b8cbfa338274df6ba9c83ee44a17f
Author: magnum <john.magnum@hushmail.com>
Date:   Mon May 6 02:09:43 2019 +0200

    CRC32C: Report AVX if applicable
    Stribog: Report AVX if applicable
    Pomelo: Report AVX/AVX2 if applicable
    
    See #3881
```
Apparently GitHub sorts by commit timestamp.